BridgeAppSDK/SBABridgeTask+Dictionary.swift

```diff
@@ -68,7 +68,7 @@ extension NSDictionary: SBABridgeTask {
     }
 
     public var schemaRevision: NSNumber! {
-        return self["schemaRevision"] as? NSNumber ?? NSNumber(integer:1)
+        return self["schemaRevision"] as? NSNumber ?? NSNumber(integer:0)
```

Comment:
?? default schema revision is 1--you seem to have changed this throughout BridgeAppSDK...?